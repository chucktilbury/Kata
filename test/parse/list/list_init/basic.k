;-------------------------------------------------------------------------------
; Test the basic initializers for an array definition.
;-------------------------------------------------------------------------------

var list name = [123, 123, 123]
var list name = [asdf.qwe.asd]
var list name = ["asdf"(123, "qwe"), sdkj, 123]
var list name = ["asdf"(123, "qwe"), asd.sdkj[1][2], 123]
