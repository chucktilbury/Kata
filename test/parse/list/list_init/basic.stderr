Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/basic.k: 8: 47: expected a ',' or a ']' but got [
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/basic.k: 8: 48: expected a dictionary init element but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/basic.k: 8: 48: expected an assignment item but got [
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/basic.k: 8: 48: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/basic.k: 8: 48: module cannot be empty
